php/types/events/register.php

```diff
@@ -44,62 +44,6 @@ function event_date_meta($event) {
     </div><?php
 }
 
-    register_rest_field('event', 'extra_info', array(
```

Comment:
Register REST fields in rest.php files next time.

Reply:
noted. [it looks like you committed this](https://github.com/ktuh-wordpress-site/custom-react-theme/commit/9a12c32c38af39efc8043baaeddbfacf3a047a55#diff-fb14ea8fb00e962eb1f2f6c380673352c2ee1066ccf94dbe2b7c7f3e1c017af7) Mar 4, 2020